### PR TITLE
Fix Configuration Layout Screen

### PR DESCRIPTION
**Pull request by FrauR**
added more space and different constraints to the feature cells in the configuration screen.

#29

### CODE REVIEW

File: Classes/FlipTheSwitch/FTSFeature.h [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- (id)init NS_UNAVAILABLE;
+- (instancetype)initWithName:(NSString *)name enabled:(BOOL)enabled featureDescription:(NSString *)featureDescription;
+
+- (BOOL)isEqual:(id)other;
+
+- (BOOL)isEqualToFeature:(FTSFeature *)feature;
+
+- (NSUInteger)hash;
+
+- (NSString *)description;
```
Comment (garriguv) [inline Classes/FlipTheSwitch/FTSFeature.h]: No need to expose these methods:
`-isEqual:`
`-isEqualToFeature:`
`-hash`
`-description`

File: Classes/FlipTheSwitch/FTSFeatureCell.h [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
 @protocol FlipTheSwitchCellDelegate <NSObject>
-- (void)flipTheSwitchCellDidToggleFeature:(FlipTheSwitchCell *)cell;
+- (void)flipTheSwitchCellDidToggleFeature:(FTSFeatureCell *)cell;
 @end
 
-@interface FlipTheSwitchCell : UITableViewCell
+@interface FTSFeatureCell : UITableViewCell
 @property (nonatomic, weak) IBOutlet UILabel *feature;
 @property (nonatomic, weak) IBOutlet UISwitch *toggle;
+@property (weak, nonatomic) IBOutlet UILabel *featureDescription;
```
Comment (garriguv) [inline Classes/FlipTheSwitch/FTSFeatureCell.h]: `(nonatomic, weak)`

**Comment by michaelengland** [comment]
Code changes look fine to me. @garriguv can you quickly check over view changes? I'm a bit lost there.

**Comment by garriguv** [comment]
:+1:

**Pull request merged**